### PR TITLE
Allow up to 128MB stack size for build.rs

### PR DESCRIPTION
**Pull request by nox**

### CODE REVIEW

File: html5ever/build.rs [hunk 1]
```diff
@@ -13,17 +13,21 @@ extern crate proc_macro2;
...
-    let rules_rs = Path::new(&manifest_dir).join("src/tree_builder/rules.rs");
-    match_token::expand(
-        &rules_rs,
-        &Path::new(&env::var("OUT_DIR").unwrap()).join("rules.rs"));
+    let input = Path::new(&manifest_dir).join("src/tree_builder/rules.rs");
+    let output = Path::new(&env::var("OUT_DIR").unwrap()).join("rules.rs");
+    println!("cargo:rerun-if-changed={}", input.display());
 
-    println!("cargo:rerun-if-changed={}", rules_rs.display());
+    let handle = Builder::new().stack_size(128 * 1024 * 1024).spawn(move || {
```
Comment (emilio) [inline html5ever/build.rs]: Add a comment?

**Comment by Ygg01** [comment]
@nox What's causing `Builder` to take so much space?

**Comment by nox** [comment]
What do you mean?

-- 
Anthony Ramine

**Comment by Ygg01** [comment]
@nox To clarify [comment](https://github.com/servo/html5ever/pull/343/files#diff-f876fb690467b5a467c5e1bb719fe817R28) it says it's `we have Stack Overflow on Servo CI`. I inferred that since you are limiting the `Builder` that's what's causing the stack overflow, and wondering in what cases does it overflow?

**Comment by nox** [comment]
I don't understand what you mean by `Builder`, given I *added* the thread builder to *avoid* a stack overflow. We don't know why that stack overflow occurs but it's probably the new syn version using more memory or something like that.

**Comment by Ygg01** [comment]
Mea culpa, I thought I saw `Builder` in old code. Anyway thanks for the response.

**Comment by nox** [comment]
You're welcome!

**Comment by SimonSapin** [comment]
I played a bit with different stack sizes on Linux x64. In debug mode, the build script fails when I take it under ~550 KB. The default stack size of macOS is apparently 512 KB. Release mode helps, I assume with inlining. The limit then is around ~130 KB.

Debug mode stack in gdb below. It looks like there is no infinite recursion bug, it’s just that `syn`’s parser is very recursive and we pass it deeply nested data.

<details>

```
#0  0x000055555587d014 in core::cell::BorrowRef::new ()
#1  0x000055555587b89b in <core::cell::RefCell<T>>::try_borrow ()
#2  0x000055555587bbfb in <core::cell::RefCell<T>>::borrow ()
#3  0x0000555555884991 in proc_macro2::imp::Term::as_str::{{closure}} ()
#4  0x0000555555892999 in <std::thread::local::LocalKey<T>>::try_with ()
#5  0x0000555555892318 in <std::thread::local::LocalKey<T>>::with ()
#6  0x0000555555884964 in proc_macro2::imp::Term::as_str ()
#7  0x000055555586c269 in proc_macro2::Term::as_str ()
#8  0x0000555555779d59 in syn::ident::parsing::<impl syn::synom::Synom for syn::ident::Ident>::parse ()
#9  0x0000555555787247 in syn::path::parsing::<impl syn::synom::Synom for syn::path::PathSegment>::parse ()
#10 0x0000555555856b86 in <syn::punctuated::Punctuated<T, P>>::parse ()
#11 0x000055555584ae47 in <syn::punctuated::Punctuated<T, P>>::parse_separated_nonempty_with ()
#12 0x000055555584aaa4 in <syn::punctuated::Punctuated<T, P>>::parse_separated_nonempty ()
#13 0x0000555555784dbc in syn::path::parsing::<impl syn::synom::Synom for syn::path::Path>::parse ()
#14 0x000055555571ea9a in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::ExprStruct>::parse ()
#15 0x00005555557f24b5 in syn::expr::parsing::atom_expr ()
#16 0x00005555557ef31d in syn::expr::parsing::trailer_expr ()
#17 0x00005555557eeba8 in syn::expr::parsing::unary_expr ()
#18 0x00005555557eb55d in syn::expr::parsing::cast_expr ()
#19 0x00005555557e9b92 in syn::expr::parsing::term_expr ()
#20 0x00005555557e8322 in syn::expr::parsing::arith_expr ()
#21 0x00005555557e6ab2 in syn::expr::parsing::shift_expr ()
#22 0x00005555557e4e52 in syn::expr::parsing::bitand_expr ()
#23 0x00005555557e346a in syn::expr::parsing::bitxor_expr ()
#24 0x00005555557e1802 in syn::expr::parsing::bitor_expr ()
#25 0x00005555557df5fa in syn::expr::parsing::compare_expr ()
#26 0x00005555557dde32 in syn::expr::parsing::and_expr ()
#27 0x00005555557dc682 in syn::expr::parsing::or_expr ()
#28 0x00005555557dadf2 in syn::expr::parsing::range_expr ()
#29 0x00005555557d9cc2 in syn::expr::parsing::placement_expr ()
#30 0x00005555557d819a in syn::expr::parsing::assign_expr ()
#31 0x00005555557d8091 in syn::expr::parsing::ambiguous_expr ()
#32 0x000055555571114f in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Expr>::parse ()
#33 0x000055555584d4e6 in <syn::punctuated::Punctuated<T, P>>::parse ()
#34 0x000055555584b2e3 in <syn::punctuated::Punctuated<T, P>>::parse_terminated_with ()
#35 0x000055555584ab04 in <syn::punctuated::Punctuated<T, P>>::parse_terminated ()
#36 0x00005555557f7e5d in syn::expr::parsing::and_call::{{closure}} ()
#37 0x00005555557a2e50 in syn::token::parsing::delim ()
#38 0x0000555555768984 in syn::token::Paren::parse ()
#39 0x00005555557f7e3d in syn::expr::parsing::and_call ()
#40 0x00005555557ef775 in syn::expr::parsing::trailer_expr ()
#41 0x00005555557eeba8 in syn::expr::parsing::unary_expr ()
#42 0x00005555557eb55d in syn::expr::parsing::cast_expr ()
#43 0x00005555557e9b92 in syn::expr::parsing::term_expr ()
#44 0x00005555557e8322 in syn::expr::parsing::arith_expr ()
#45 0x00005555557e6ab2 in syn::expr::parsing::shift_expr ()
#46 0x00005555557e4e52 in syn::expr::parsing::bitand_expr ()
#47 0x00005555557e346a in syn::expr::parsing::bitxor_expr ()
#48 0x00005555557e1802 in syn::expr::parsing::bitor_expr ()
#49 0x00005555557df5fa in syn::expr::parsing::compare_expr ()
#50 0x00005555557dde32 in syn::expr::parsing::and_expr ()
#51 0x00005555557dc682 in syn::expr::parsing::or_expr ()
#52 0x00005555557dadf2 in syn::expr::parsing::range_expr ()
#53 0x00005555557d9cc2 in syn::expr::parsing::placement_expr ()
#54 0x00005555557d819a in syn::expr::parsing::assign_expr ()
#55 0x00005555557d8091 in syn::expr::parsing::ambiguous_expr ()
#56 0x000055555571114f in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Expr>::parse ()
#57 0x0000555555719fb7 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Arm>::parse ()
#58 0x0000555555778172 in syn::parsers::many0 ()
#59 0x00005555557fa634 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::ExprMatch>::parse::{{closure}} ()
#60 0x00005555557a96f0 in syn::token::parsing::delim ()
#61 0x0000555555767764 in syn::token::Brace::parse ()
#62 0x0000555555717644 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::ExprMatch>::parse ()
#63 0x00005555557f6859 in syn::expr::parsing::expr_nosemi ()
#64 0x000055555580617f in syn::expr::parsing::stmt_blockexpr ()
#65 0x000055555572561b in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Stmt>::parse ()
#66 0x0000555555722f0b in syn::expr::parsing::<impl syn::expr::Block>::parse_within ()
#67 0x000055555580239d in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Block>::parse::{{closure}} ()
#68 0x000055555579aa80 in syn::token::parsing::delim ()
#69 0x0000555555767624 in syn::token::Brace::parse ()
#70 0x0000555555722461 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Block>::parse ()
#71 0x00005555557217f1 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::ExprBlock>::parse ()
#72 0x00005555557f4750 in syn::expr::parsing::atom_expr ()
#73 0x00005555557ef31d in syn::expr::parsing::trailer_expr ()
#74 0x00005555557eeba8 in syn::expr::parsing::unary_expr ()
#75 0x00005555557eb55d in syn::expr::parsing::cast_expr ()
#76 0x00005555557e9b92 in syn::expr::parsing::term_expr ()
#77 0x00005555557e8322 in syn::expr::parsing::arith_expr ()
#78 0x00005555557e6ab2 in syn::expr::parsing::shift_expr ()
#79 0x00005555557e4e52 in syn::expr::parsing::bitand_expr ()
#80 0x00005555557e346a in syn::expr::parsing::bitxor_expr ()
#81 0x00005555557e1802 in syn::expr::parsing::bitor_expr ()
#82 0x00005555557df5fa in syn::expr::parsing::compare_expr ()
#83 0x00005555557dde32 in syn::expr::parsing::and_expr ()
#84 0x00005555557dc682 in syn::expr::parsing::or_expr ()
#85 0x00005555557dadf2 in syn::expr::parsing::range_expr ()
#86 0x00005555557d9cc2 in syn::expr::parsing::placement_expr ()
#87 0x00005555557d819a in syn::expr::parsing::assign_expr ()
#88 0x00005555557d8091 in syn::expr::parsing::ambiguous_expr ()
#89 0x00005555557fbac4 in syn::expr::parsing::expr_closure ()
#90 0x00005555557f44bc in syn::expr::parsing::atom_expr ()
#91 0x00005555557ef31d in syn::expr::parsing::trailer_expr ()
#92 0x00005555557eeba8 in syn::expr::parsing::unary_expr ()
#93 0x00005555557eb55d in syn::expr::parsing::cast_expr ()
#94 0x00005555557e9b92 in syn::expr::parsing::term_expr ()
#95 0x00005555557e8322 in syn::expr::parsing::arith_expr ()
#96 0x00005555557e6ab2 in syn::expr::parsing::shift_expr ()
#97 0x00005555557e4e52 in syn::expr::parsing::bitand_expr ()
#98 0x00005555557e346a in syn::expr::parsing::bitxor_expr ()
#99 0x00005555557e1802 in syn::expr::parsing::bitor_expr ()
#100 0x00005555557df5fa in syn::expr::parsing::compare_expr ()
#101 0x00005555557dde32 in syn::expr::parsing::and_expr ()
#102 0x00005555557dc682 in syn::expr::parsing::or_expr ()
#103 0x00005555557dadf2 in syn::expr::parsing::range_expr ()
#104 0x00005555557d9cc2 in syn::expr::parsing::placement_expr ()
#105 0x00005555557d819a in syn::expr::parsing::assign_expr ()
#106 0x00005555557d8091 in syn::expr::parsing::ambiguous_expr ()
#107 0x000055555571114f in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Expr>::parse ()
#108 0x000055555584d4e6 in <syn::punctuated::Punctuated<T, P>>::parse ()
#109 0x000055555584b2e3 in <syn::punctuated::Punctuated<T, P>>::parse_terminated_with ()
#110 0x000055555584ab04 in <syn::punctuated::Punctuated<T, P>>::parse_terminated ()
#111 0x00005555557f956d in syn::expr::parsing::and_method_call::{{closure}} ()
#112 0x000055555579cb50 in syn::token::parsing::delim ()
#113 0x00005555557685c4 in syn::token::Paren::parse ()
#114 0x00005555557f8a4d in syn::expr::parsing::and_method_call ()
#115 0x00005555557efc5d in syn::expr::parsing::trailer_expr ()
#116 0x00005555557eeba8 in syn::expr::parsing::unary_expr ()
#117 0x00005555557eb55d in syn::expr::parsing::cast_expr ()
#118 0x00005555557e9b92 in syn::expr::parsing::term_expr ()
#119 0x00005555557e8322 in syn::expr::parsing::arith_expr ()
#120 0x00005555557e6ab2 in syn::expr::parsing::shift_expr ()
#121 0x00005555557e4e52 in syn::expr::parsing::bitand_expr ()
#122 0x00005555557e346a in syn::expr::parsing::bitxor_expr ()
#123 0x00005555557e1802 in syn::expr::parsing::bitor_expr ()
#124 0x00005555557df5fa in syn::expr::parsing::compare_expr ()
#125 0x00005555557dde32 in syn::expr::parsing::and_expr ()
#126 0x00005555557dc682 in syn::expr::parsing::or_expr ()
#127 0x00005555557dadf2 in syn::expr::parsing::range_expr ()
#128 0x00005555557d9cc2 in syn::expr::parsing::placement_expr ()
#129 0x00005555557d819a in syn::expr::parsing::assign_expr ()
#130 0x00005555557d8091 in syn::expr::parsing::ambiguous_expr ()
#131 0x000055555571114f in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Expr>::parse ()
#132 0x0000555555804444 in syn::expr::parsing::stmt_local ()
#133 0x000055555572543f in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Stmt>::parse ()
#134 0x0000555555722f0b in syn::expr::parsing::<impl syn::expr::Block>::parse_within ()
#135 0x000055555580239d in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Block>::parse::{{closure}} ()
#136 0x000055555579aa80 in syn::token::parsing::delim ()
#137 0x0000555555767624 in syn::token::Brace::parse ()
#138 0x0000555555722461 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Block>::parse ()
#139 0x00005555557217f1 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::ExprBlock>::parse ()
#140 0x00005555557f7011 in syn::expr::parsing::expr_nosemi ()
#141 0x0000555555719eb1 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Arm>::parse ()
#142 0x0000555555778172 in syn::parsers::many0 ()
#143 0x00005555557fa634 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::ExprMatch>::parse::{{closure}}
    ()
#144 0x00005555557a96f0 in syn::token::parsing::delim ()
#145 0x0000555555767764 in syn::token::Brace::parse ()
#146 0x0000555555717644 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::ExprMatch>::parse ()
#147 0x00005555557f6859 in syn::expr::parsing::expr_nosemi ()
#148 0x000055555580617f in syn::expr::parsing::stmt_blockexpr ()
#149 0x000055555572561b in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Stmt>::parse ()
#150 0x0000555555722f0b in syn::expr::parsing::<impl syn::expr::Block>::parse_within ()
#151 0x000055555580239d in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Block>::parse::{{closure}} ()
#152 0x000055555579aa80 in syn::token::parsing::delim ()
#153 0x0000555555767624 in syn::token::Brace::parse ()
#154 0x0000555555722461 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Block>::parse ()
#155 0x00005555557217f1 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::ExprBlock>::parse ()
#156 0x00005555557f7011 in syn::expr::parsing::expr_nosemi ()
#157 0x0000555555719eb1 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Arm>::parse ()
#158 0x0000555555778172 in syn::parsers::many0 ()
#159 0x00005555557fa634 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::ExprMatch>::parse::{{closure}}
    ()
#160 0x00005555557a96f0 in syn::token::parsing::delim ()
#161 0x0000555555767764 in syn::token::Brace::parse ()
#162 0x0000555555717644 in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::ExprMatch>::parse ()
#163 0x00005555557f3d4c in syn::expr::parsing::atom_expr ()
#164 0x00005555557ef31d in syn::expr::parsing::trailer_expr ()
#165 0x00005555557eeba8 in syn::expr::parsing::unary_expr ()
#166 0x00005555557eb55d in syn::expr::parsing::cast_expr ()
#167 0x00005555557e9b92 in syn::expr::parsing::term_expr ()
#168 0x00005555557e8322 in syn::expr::parsing::arith_expr ()
#169 0x00005555557e6ab2 in syn::expr::parsing::shift_expr ()
#170 0x00005555557e4e52 in syn::expr::parsing::bitand_expr ()
#171 0x00005555557e346a in syn::expr::parsing::bitxor_expr ()
#172 0x00005555557e1802 in syn::expr::parsing::bitor_expr ()
#173 0x00005555557df5fa in syn::expr::parsing::compare_expr ()
#174 0x00005555557dde32 in syn::expr::parsing::and_expr ()
#175 0x00005555557dc682 in syn::expr::parsing::or_expr ()
#176 0x00005555557dadf2 in syn::expr::parsing::range_expr ()
#177 0x00005555557d9cc2 in syn::expr::parsing::placement_expr ()
#178 0x00005555557d819a in syn::expr::parsing::assign_expr ()
#179 0x00005555557d8091 in syn::expr::parsing::ambiguous_expr ()
#180 0x000055555571114f in syn::expr::parsing::<impl syn::synom::Synom for syn::expr::Expr>::parse ()
#181 0x00005555555c9414 in core::ops::function::FnOnce::call_once ()
#182 0x00005555556b394f in <F as syn::synom::Parser>::parse2 ()
#183 0x0000555555681b0e in syn::parse2 ()
#184 0x0000555555674e75 in build_script_build::match_token::expand_match_token ()
#185 0x00005555556790fe in <build_script_build::match_token::MatchTokenParser as syn::gen::fold::Fold>::fold_expr ()
#186 0x00005555556a8b89 in syn::gen::fold::fold_arm ()
#187 0x0000555555643f2c in syn::gen::fold::Fold::fold_arm ()
#188 0x0000555555697192 in syn::gen::fold::fold_expr_match::{{closure}} ()
#189 0x00005555556b3280 in core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &'a mut F>::call_once
    ()
#190 0x00005555556678a4 in <core::option::Option<T>>::map ()
#191 0x000055555560e974 in <core::iter::Map<I, F> as core::iter::iterator::Iterator>::next ()
#192 0x000055555559fded in <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T, I>>::spec_extend ()
#193 0x00005555555c147f in <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T, I>>::from_iter ()
#194 0x00005555555c541a in <alloc::vec::Vec<T> as core::iter::traits::FromIterator<T>>::from_iter ()
#195 0x0000555555608c9a in core::iter::iterator::Iterator::collect ()
#196 0x0000555555597988 in <alloc::vec::Vec<T> as syn::gen::helper::fold::FoldHelper>::lift ()
#197 0x000055555569702b in syn::gen::fold::fold_expr_match ()
#198 0x0000555555641ecc in syn::gen::fold::Fold::fold_expr_match ()
#199 0x00005555556ab385 in syn::gen::fold::fold_expr ()
#200 0x000055555567900b in <build_script_build::match_token::MatchTokenParser as syn::gen::fold::Fold>::fold_expr ()
#201 0x00005555556af2fd in syn::gen::fold::fold_stmt ()
#202 0x0000555555678cfc in <build_script_build::match_token::MatchTokenParser as syn::gen::fold::Fold>::fold_stmt ()
#203 0x000055555568c237 in syn::gen::fold::fold_block::{{closure}} ()
#204 0x00005555556b0df0 in core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &'a mut F>::call_once
    ()
#205 0x00005555556653cf in <core::option::Option<T>>::map ()
#206 0x000055555560eb56 in <core::iter::Map<I, F> as core::iter::iterator::Iterator>::next ()
#207 0x00005555555abd88 in <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T, I>>::spec_extend ()
#208 0x00005555555c1f1f in <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T, I>>::from_iter ()
#209 0x00005555555c54fa in <alloc::vec::Vec<T> as core::iter::traits::FromIterator<T>>::from_iter ()
#210 0x0000555555608fba in core::iter::iterator::Iterator::collect ()
#211 0x0000555555596e48 in <alloc::vec::Vec<T> as syn::gen::helper::fold::FoldHelper>::lift ()
#212 0x000055555568c170 in syn::gen::fold::fold_block ()
#213 0x0000555555640b03 in syn::gen::fold::Fold::fold_block ()
#214 0x00005555556a5f09 in syn::gen::fold::fold_impl_item_method ()
#215 0x000055555564385f in syn::gen::fold::Fold::fold_impl_item_method ()
#216 0x0000555555693052 in syn::gen::fold::fold_impl_item ()
#217 0x00005555556417b1 in syn::gen::fold::Fold::fold_impl_item ()
#218 0x0000555555693fa7 in syn::gen::fold::fold_item_impl::{{closure}} ()
#219 0x00005555556b2aa0 in core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &'a mut F>::call_once
    ()
#220 0x0000555555654b4f in <core::option::Option<T>>::map ()
#221 0x000055555560e366 in <core::iter::Map<I, F> as core::iter::iterator::Iterator>::next ()
#222 0x00005555555b65a8 in <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T, I>>::spec_extend ()
#223 0x00005555555c169f in <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T, I>>::from_iter ()
#224 0x00005555555c5bfa in <alloc::vec::Vec<T> as core::iter::traits::FromIterator<T>>::from_iter ()
#225 0x0000555555609b9a in core::iter::iterator::Iterator::collect ()
#226 0x0000555555594d48 in <alloc::vec::Vec<T> as syn::gen::helper::fold::FoldHelper>::lift ()
#227 0x0000555555693cfe in syn::gen::fold::fold_item_impl ()
#228 0x0000555555641871 in syn::gen::fold::Fold::fold_item_impl ()
#229 0x00005555556ae2fa in syn::gen::fold::fold_item ()
#230 0x000055555564409f in syn::gen::fold::Fold::fold_item ()
#231 0x00005555556ad785 in syn::gen::fold::fold_file::{{closure}} ()
#232 0x00005555556b2470 in core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &'a mut F>::call_once
    ()
#233 0x000055555566087f in <core::option::Option<T>>::map ()
#234 0x000055555560fd07 in <core::iter::Map<I, F> as core::iter::iterator::Iterator>::next ()
#235 0x00005555555ba918 in <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T, I>>::spec_extend ()
#236 0x00005555555bf9ef in <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T, I>>::from_iter ()
#237 0x00005555555c3f8a in <alloc::vec::Vec<T> as core::iter::traits::FromIterator<T>>::from_iter ()
#238 0x0000555555609c3a in core::iter::iterator::Iterator::collect ()
#239 0x0000555555595ac8 in <alloc::vec::Vec<T> as syn::gen::helper::fold::FoldHelper>::lift ()
#240 0x00005555556ad5d9 in syn::gen::fold::fold_file ()
#241 0x000055555564403c in syn::gen::fold::Fold::fold_file ()
#242 0x0000555555674a4d in build_script_build::match_token::expand ()
#243 0x000055555560581a in build_script_build::main::{{closure}} ()
#244 0x0000555555680484 in std::sys_common::backtrace::__rust_begin_short_backtrace ()
#245 0x000055555556eb14 in std::thread::Builder::spawn::{{closure}}::{{closure}} ()
#246 0x0000555555623282 in <std::panic::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once ()
#247 0x00005555555ef78f in std::panicking::try::do_call ()
#248 0x00007ffff78f632f in __rust_maybe_catch_panic () at libpanic_unwind/lib.rs:102
#249 0x00005555555ef692 in std::panicking::try ()
#250 0x00005555556232c8 in std::panic::catch_unwind ()
#251 0x000055555556e92a in std::thread::Builder::spawn::{{closure}} ()
#252 0x000055555556ec4b in <F as alloc::boxed::FnBox<A>>::call_box ()
#253 0x00007ffff78ed58c in _$LT$alloc..boxed..Box$LT$alloc..boxed..FnBox$LT$A$C$$u20$Output$u3d$R$GT$$u20$$u2b$$u20$$u27$a$GT$$u20$as$u20$core..ops..function..FnOnce$LT$A$GT$$GT$::call_once::h598f9713c9cb9093 ()
    at /checkout/src/liballoc/boxed.rs:798
#254 std::sys_common::thread::start_thread () at libstd/sys_common/thread.rs:24
#255 std::sys::unix::thread::Thread::new::thread_start () at libstd/sys/unix/thread.rs:90
#256 0x00007ffff765a0bc in start_thread () from /usr/lib/libpthread.so.0
#257 0x00007ffff71782ff in clone () from /usr/lib/libc.so.6
```

</details>

**Pull request merged**